Calculator/Calculator/CalculatorViewController.swift

```diff
@@ -132,65 +159,53 @@ class CalculatorViewController: UIViewController {
             }
             
             calculationFormula += " " + convertOperator(operatorValue)
-        } else {
-            guard let operatorText = inputOperatorLabel.text,
-                  let doubleValue = numberFormatter.string(for: Double(numberWithOutComma)) else { return }
-            
-            addStackView(operatorText: operatorText, inputText: doubleValue)
+        default:
+            addStackView(numberWithOutComma)
             calculationFormula += " " + numberWithOutComma
             calculationFormula += " " + convertOperator(operatorValue)
             inputNumberLabel.text = Literal.numberZero.value
             inputOperatorLabel.text = operatorValue
         }
     }
     
-    private func showResult(inputText: String) {
-        let numberWithOutComma = inputText.replacingOccurrences(of: Literal.comma.value, with: "")
+    private func convertOperator(_ operatorValue: String) -> String {
+        switch operatorValue {
+        case Literal.multiplication.value:
+            return Literal.realMultiplication.value
+        case Literal.division.value:
+            return Literal.realDivision.value
+        default:
+            return operatorValue
+        }
+    }
+    
+    private func checkOperatorCount() -> Bool {
         let operators = Operator.allCases.map { convertOperator(String($0.rawValue)) }.joined()
         let separators = CharacterSet(charactersIn: operators)
         
         if calculationFormula.components(separatedBy: separators).count <= 1 {
-            return
+            return false
         }
+        return true
+    }
+    
+    private func showResult(inputText: String) {
+        guard checkOperatorCount() == true else { return }
```

Comment:
`checkOperatorCount()`가 `Bool?` 을 리턴하는 것이 아니기 때문에 
`== true` 제거해주어도 됩니다 ㅎㅎ 
```suggestion
        guard checkOperatorCount() else { return }
```